### ISSUE TITLE
unzip is required to install bun (see: https://github.com/oven-sh/bun#unzip-is-required)

### ISSUE THREAD

**Issue by coding-agent** [oven-sh/bun#4699]
### What platform is your computer?

os: win32, arch: x64

### What steps can reproduce the bug?

I'm using Ubuntu wsl:
```shell
wsl -d ubuntu
``` 
```shell
curl -fsSL https://bun.sh/install | bash`
``` 

### What do you see instead?

error: unzip is required to install bun (see: https://github.com/oven-sh/bun#unzip-is-required)

### Additional information

I couldn't find any information for the "unzip-is-required" in the repo

**Comment by GitHubFoxy** [comment]
U should update linux packages, install unzip by hand. 
sudo apt update
sudo apt-get install unzip
Im using windows 10, worked for me

**Comment by viveleroi** [comment]
Just encountered this too. Just so the issue filed is clear, the URL bun prints to the console (https://github.com/oven-sh/bun#unzip-is-required) does not lead to anything inside the README of this repo, so it's a bad URL.

The apt update/install unzip steps do work to resolve the unzip issue, though.

**Comment by Fiontan** [comment]
Also having a go on windows 10 wsl and the docs are clearISH, 


after the initial install steps (enabling WSL and installing a distro) you need too run sudo apt install unzip. The curl will run as expected.

**Comment by coding-agent** [comment]
@GitHubFoxy Thank you, worked for me too 🚀🔥
 And Thank you @viveleroi and @Fiontan too

**Comment by blowsie** [comment]
The cli still points you to the wrong place.

I think the URL should be 
https://github.com/oven-sh/bun/blob/main/docs/troubleshooting.md#unzip-is-required

**Comment by Fiewor** [comment]
> The cli still points you to the wrong place.
> 
> I think the URL should be https://github.com/oven-sh/bun/blob/main/docs/troubleshooting.md#unzip-is-required

I just ran into the same error and it still points to the wrong URL

**Comment by Yay295** [comment]
It looks like this was fixed with #2669, and then removed with #3389, but the website is still serving the version from Feb 25, 2023.

**Comment by zonglin-hub** [comment]
```sh
~> cat /proc/version
Linux version 5.15.0-78-generic (buildd@lcy02-amd64-008) (gcc (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0, GNU ld (GNU Binutils for Ubuntu) 2.38) #85-Ubuntu SMP Fri Jul 7 15:25:09 UTC 2023
~> curl -fsSL https://bun.sh/install | bash
error: unzip is required to install bun (see: https://bun.sh/docs/installation#macos-and-linux)
```
I need an effective command.
Avoid unnecessary manual operations.
